insonmnia/logging/logging_test.go

```
@@ -0,0 +1,28 @@
+package logging
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap/zapcore"
+)
+
+func TestParseLogLevel(t *testing.T) {
+	tests := []struct {
+		in  string
+		out zapcore.Level
+	}{
+		// any case must be parsed
+		{in: "warn", out: zapcore.WarnLevel},
+		{in: "Warn", out: zapcore.WarnLevel},
+		{in: "WARN", out: zapcore.WarnLevel},
+		// any other values must be represented as default debug level
+		{in: "-1", out: zapcore.DebugLevel},
```

It is surprising for users. I'd rather throw an error on other values.